modules/porous_flow/src/actions/PorousFlowActionBase.C

```diff
@@ -236,6 +257,8 @@ PorousFlowActionBase::addSaturationAux(unsigned phase)
     InputParameters params = _factory.getValidParams(aux_kernel_type);
 
     std::string aux_kernel_name = "PorousFlowActionBase_SaturationAux" + phase_str;
+    if (_subdomain_names_set)
+      params.set<std::vector<SubdomainName>>("block") = _subdomain_names;
```

Comment:
same for all auxkernels (when the variable is one added by the action, with the block restriction)